lua/nvim-treesitter/locals.lua

```diff
@@ -177,7 +195,7 @@ function M.containing_scope(node, bufnr)
     iter_node = iter_node:parent()
   end
 
-  return iter_node or node
+  return iter_node or nil
```

Comment:
I needed this change in order to be able to determine between a scope being found verses the node being a scope. I update all usages to account for this.

Reply:
You could add third parameter `allow_scope` or something similar. if `nil` or `true`, then you return the `node`, else you just retun `nil` at the end.

Reply:
(and you could call it with `false` passed)

Reply:
I like this. I'll update it.